packer/plugin_folders.go

```diff
@@ -4,16 +4,24 @@
 package packer
 
 import (
+	"fmt"
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/hashicorp/packer-plugin-sdk/pathing"
 )
 
 // PluginFolder returns the known plugin folder based on system.
 func PluginFolder() (string, error) {
 	if packerPluginPath := os.Getenv("PACKER_PLUGIN_PATH"); packerPluginPath != "" {
+		if strings.Contains(packerPluginPath, ":") {
```

Comment:
The operator on windows is ";" you can use os.PathListSeparator here to determine if there are multiple paths defined.